packages/stream-server/stream_server.js

```diff
@@ -27,6 +27,16 @@ var websocketExtensions = _.once(function () {
 
 var pathPrefix = __meteor_runtime_config__.ROOT_URL_PATH_PREFIX ||  "";
 
+/**
+ * List of stream servers
+ *
+ * DDP LiveData server will load the latest added stream server.
+ * You need place custom stream server package in `.meteor/packages`
+ * before packages `meteor-tools`, ddp` or `ddp-server`
+ * @type {(StreamServer|*)[]}
+ */
+StreamServers = [];
```

Comment:
Per my previous comments, this should become something like
```js
export function addCustomStreamServer(server) { ... }
```

Reply:
I don't found the more simple and elegant way of adding a custom stream server into the Meteor. 

![2f8g49](https://user-images.githubusercontent.com/2821574/43656726-1c240e92-975c-11e8-9778-195f1be9f442.jpg)

My solution is:

1. Create the `StreamServers` array in the base package `stream-server`.
2. Place `custom-stream-server` before package that loading `ddp-server`.
3. In the `custom-stream-server` `.use()` and `.imply()` `stream-server`, then push the new stream server to the `StreamServers` array.
4. Get the last added stream server  in `livedata_server.js`.

Maybe, I don't know the right way changing of DDP stream server on the fly.

@benjamn for what task we trying to find the solution:
a) add the `stream-server-uws` package as a weak dependency to the `ddp-server`;
b) make the `stream-server` changeable?